### ISSUE TITLE
docs: add more package executers

### ISSUE THREAD

**Issue by Jay-Karia** [facebook/docusaurus#10355]
### Have you read the Contributing Guidelines on issues?

- [X] I have read the [Contributing Guidelines on issues](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#issues).

### Description

In this [page](https://docusaurus.io/docs/typescript-support#initialization), the initialization of the project with typescript has the following command: `npx create-docusaurus@latest my-website classic --typescript`, it is using `npx`, we can add other package executors for `yarn` and `pnpm`.

Such as:
yarn: `yarn dlx`
pnpm: `pnpm dlx`.

Other pages with the same issue:
https://docusaurus.io/docs/i18n/git#initialization
https://docusaurus.io/blog/releases/3.0#query-string-data-attributes
https://docusaurus.io/docs#fast-track
https://docusaurus.io/docs/next/installation#scaffold-project-website

### Self-service

- [X] I'd be willing to address this documentation request myself.

**Comment by slorber** [comment]
Yes, we could, although it's not a significant win since users can still use their preferred package manager afterward.

Also, this requires us to use tabs for package managers now, and I don't think the remark plugin we use supports npx to yarn/pnpm conversions

https://github.com/nebrelbug/npm-to-yarn

**Comment by Jay-Karia** [comment]
Submitted an issue for a new feautre to the plugin used.

Will work on this.

**Comment by slorber** [comment]
Thanks, we'll track that issue: https://github.com/nebrelbug/npm-to-yarn/issues/48

**Comment by dibyajyoti-mandal** [comment]
May i work on this issue?

**Comment by slorber** [comment]
I'd prefer to work on this myself later. I'm not sure we want to present package manager tabs as a first encounter to new users. Sometimes having less choice is a feature, not a bug. 

A choice implies an extra cognitive load so we should probably use this sparingly.

On their first encounter with Docusaurus, users not familiar with the Node.js ecosystem should rather not have to make any decision, and just execute commands we give them directly.

**Comment by LiuHuaize** [comment]
I am working on this.
